src/shared/components/Header/index.jsx

```diff
@@ -69,7 +69,7 @@ const Header = ({
 
   if (TopNavRef) {
     return (
-      <div>
+      <div style={{ zIndex: 2 }}>
```

Comment:
@gets0ul Please don't use inline styles, move this to .scss file.

Reply:
@luizrrodrigues Done.